scripts/build

```diff
@@ -265,6 +264,36 @@ if [ ! -f $STAMP ]; then
         PKG_MESON_SCRIPT="$PKG_BUILD/meson.build"
       fi
 
+      # auto detect toolchain
+      _auto_toolchain=""
+      if [ -z "$PKG_TOOLCHAIN" -o "$PKG_TOOLCHAIN" = "auto" ]; then
+        if [ -f "$PKG_MESON_SCRIPT" ]; then
+          PKG_TOOLCHAIN="meson"
+        elif [ -f "$PKG_CMAKE_SCRIPT" -a ! "$PKG_USE_CMAKE" = "no" ]; then
+          PKG_TOOLCHAIN="cmake"
+        elif [ -f "$PKG_CONFIGURE_SCRIPT" ]; then
+          PKG_TOOLCHAIN="configure"
+        elif [ -f $PKG_BUILD/Makefile ]; then
+          PKG_TOOLCHAIN="make"
+        else
+          echo "Not possible to detect toolchain automactcly. Add PKG_TOOLCHAIN= in to package.mk"
```

Comment:
Typo + wording:

echo "Not possible to detect toolchain automatically. Add PKG_TOOLCHAIN= to package.mk"

Reply:
... c++ is much more easier then english :)